react/BaseContent.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { ExtensionPoint } from 'vtex.render-runtime'
+import { useOrderForm } from 'vtex.order-manager/OrderForm'
+import { useCssHandles } from 'vtex.css-handles'
+import { Button } from 'vtex.styleguide'
+
+import { useMinicartState } from './MinicartContext'
+
+interface OrderFormContext {
+  orderForm: OrderForm
+  loading: boolean
+}
+
+interface Props {
+  sideBarMode: boolean
+  finishShoppingButtonLink: string
+}
+
+const CSS_HANDLES = [
+  'minicartContent',
+  'minicartFooter',
+  'minicartEmptyStateText',
+  'minicartEmptyStateContainer',
+] as const
+
+const Content: FC<Props> = ({ finishShoppingButtonLink }) => {
+  const { orderForm, loading }: OrderFormContext = useOrderForm()
+  const handles = useCssHandles(CSS_HANDLES)
+  const { variation } = useMinicartState()
+
+  const sideBarStyles = {
+    height: '100%',
+  }
+
+  const popupStyles = {
+    maxHeight: 600,
+  }
+
+  return !loading && orderForm.items.length === 0 ? (
+    <div
+      className={`${handles.minicartEmptyStateContainer} pa9 flex justify-center`}
+      style={{ width: 340 }}
+    >
+      <span className={`${handles.minicartEmptyStateText} t-body`}>
+        <FormattedMessage id="store/minicart.empty-state" />
+      </span>
+    </div>
+  ) : (
+    <div
+      className={`${handles.minicartContent} flex flex-column justify-between`}
+      style={variation === 'drawer' ? sideBarStyles : popupStyles}
+    >
+      <div className="w-100 overflow-y-scroll">
+        <h3 className="t-heading-3 mv2 c-on-base">Cart</h3>
+        <ExtensionPoint id="minicart-product-list" />
+      </div>
+      <div
+        className={`${handles.minicartFooter} ${
+          variation === 'drawer' ? 'pa4' : 'pv3'
+        } sticky`}
```

Comment:
Why don't you use `classNames` here?

Reply:
I really dislike using `classnames` because I think that just adds an unnecessary package to the project. I know that this repo already uses it because the legacy Minicart uses `classnames`, but I chose to avoid using it in new code since you can achieve the exact same result by using template strings and the complexity is pretty much the same

Reply:
I don't think `classNames` are unnecessary. This is a package with just a function, so the bundle size is not that big (there are other options that do the same thing with less bundle size, clsx, but to add this is too complicated). The code could be simpler just by using `classNames` I think. Another point is the usage of template strings inside the "HTML" of the component, I don't like this because it adds logic to a part of the component that could just display things, so what you could do is create this classes outside of the return. About removing the `classNames` I think is really close to impossible considering the number of apps that have it as a dependency

Reply:
I'm going to meet you halfway. 
Not going to use `classNames`, but I'll remove the logic from the return value and store them in variables